Temperature.java

```diff
@@ -0,0 +1,28 @@
+class Temperature {
```

Comment:
Can you please leave a comment here on what we were trying to achieve here ?  As in 

// This class is an example of factory method pattern.

Reply:
Makes sense, Added here 0777280